source/command-monitoring/tests/command.json

```diff
@@ -15,6 +16,11 @@
         "arguments": {
           "filter": {
             "_id": 1
+          },
```

Comment:
I'm not clear what the purpose of this addition is.  Isn't this covered by 'read_preference' above?